web/src/app/modules/shared/components/presentation/link/link.component.html

```diff
@@ -1,5 +1,5 @@
 <ng-template [ngIf]="isExternal" [ngIfElse]="internal">
-  <a href="{{ ref }}">{{ value }}</a>
+  <a target ="_blank" href="{{ ref }}">{{ value }}</a>
```

Comment:
Let's make sure we test this with a plugin and make sure that changing the target here doesn't impact the expected behavior.

Reply:
Verified that both internal and external links work as expected inside my test plugin. Also tested that all links are working as expected inside existing plugins `knative` and `Jenkins X`